repair/row_level.cc

```diff
@@ -1269,12 +1269,14 @@ class repair_meta {
             // and _working_row_buf are ordered.
             _working_row_buf = utils::stall_free_merge(tmp, row_diff,
                  [this] (const repair_row& x, const repair_row& y) { return _cmp(x.boundary(), y.boundary()) < 0; });
+            utils::stall_free_clear(tmp);
```

Comment:
With merge_to() you can drop the tmp.swap() call, and then `merge_to(_working_row_bug, row_diff)`. There won't be a tmp that needs clearing.

I think that even with the current code tmp is empty after the merge.

Reply:
Yes.